src/views/main/components/SensorDetails.js

```diff
@@ -181,8 +190,16 @@ class SensorDetails extends Component {
 					<FaRegTimesCircle className="closeIcon" size={22}></FaRegTimesCircle>
 				</Button>
 				</div>
-				<div className="stationInside">
 				<ScrollBar className="stationList" options={{suppressScrollX : true}}>
+				<div className="close close-mobile">
+				<Button onClick={() => {
+					this.props.dispatch(sensorDetailAction(null));
+					this.setState({ visible: false });
+				}}>
+					
+					<FaRegTimesCircle className="closeIcon" size={22}></FaRegTimesCircle>
```

Comment:
What do you think? Should we change static css class name to react's style import (`style.className`)?

Reply:
We should :D